theano/tensor/elemwise.py

```diff
@@ -1300,11 +1311,10 @@ class CAReduce(Op):
     CAReduce(mul) -> product
     CAReduce(maximum) -> max
     CAReduce(minimum) -> min
-    CAReduce(or_) -> any # not lazy
-    CAReduce(and_) -> all # not lazy
+    CAReduce(or) -> any # not lazy
+    CAReduce(and) -> all # not lazy
```

Comment:
This is wrong. The underscore should probably be escaped with \ but should not be removed.